packages/screens/Mini/Network/ChangeNetworkScreen.tsx

```diff
@@ -28,20 +33,22 @@ export default function ChangeNetworkScreen({
   navigation,
 }: ChangeNetworkScreenProps) {
   const goBackTo = () => navigation.replace("MiniSettings");
+  const selectedWallet = useSelectedNativeWallet();
 
   const addresses: ChangeNetworkType[] = [
```

Comment:
```ts
const enabledNetworks = useEnabledNetworks()
```
and
```suggestion
  const addresses: ChangeNetworkType[] = enabledNetworks.map(n => ({
    id: n.id,
    label: n.displayName,
    url: n.rpcEndpoint
  }))
```
you could add a filter if needed

Reply:
thanks for your suggestion.
I tried to do it this way, but in this case doesn't make sense, because of this line.

https://github.com/TERITORI/teritori-dapp/blob/68f9e795d80541347a215c49e350ee678079486a/packages/hooks/useEnabledNetworks.ts#L12
The mini mode doesn't have an "enable testnet" setting.

The changes already in place are the toggle in mini mode at least.

It makes is more complex for no added benefit. (the mobile wallet only supports teritori at this point)

Reply:
also I don't think we should use an intermediary network reprensentation (eg ChangeNetworkType)

Reply:
don't blindly follow the figma, konolee doesn't know everything about our architecture and so some things might not match